packages/backend/src/services/ProjectService/formatRows.js

```diff
@@ -0,0 +1,40 @@
+
+const { workerData, parentPort } = require("worker_threads");
+
+const { formatItemValue } = require("@lightdash/common");
```

Comment:
I'm not sure if there is a better way to do the import here... unfortunatelly this method is too complex to just copy it here

Reply:
btw, worker_thread only allows simple values, we can't pass functions